<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelImage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cousine</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Откройте изображение...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="sliderBlurRadius">
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonOpen">
     <property name="font">
      <font>
       <family>Cousine</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Открыть</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonSave">
       <property name="font">
        <font>
         <family>Cousine</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSaveAll">
       <property name="font">
        <font>
         <family>Cousine</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Сохранить все</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="buttonPrevious">
       <property name="font">
        <font>
         <family>Cousine</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNext">
       <property name="font">
        <font>
         <family>Cousine</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOpen</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderBlurRadius</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWidget</receiver>
   <slot>setBluredImage(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSave</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSaveAll</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>saveAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonPrevious</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>previous()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>522</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonNext</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>538</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open()</slot>
  <slot>save()</slot>
  <slot>saveAll()</slot>
  <slot>previous()</slot>
  <slot>next()</slot>
  <slot>setBluredImage(int)</slot>
 </slots>
</ui>
